search/search-domain/src/main/java/io/camunda/search/filter/UserTaskFilter.java

```diff
@@ -32,10 +34,14 @@ public record UserTaskFilter(
     List<VariableValueFilter> processInstanceVariableFilter,
     List<VariableValueFilter> localVariableFilters,
     List<Long> elementInstanceKeys,
+    List<Operation<OffsetDateTime>> creationDateOperations,
+    List<Operation<OffsetDateTime>> completionDateOperations,
     String type)
     implements FilterBase {
 
   public static final class Builder implements ObjectBuilder<UserTaskFilter> {
+    List<Operation<OffsetDateTime>> creationDateOperations;
+    List<Operation<OffsetDateTime>> completionDateOperations;
```

Comment:
🔧 Let's be consistent in the code
```suggestion
    private List<Operation<OffsetDateTime>> creationDateOperations;
    private List<Operation<OffsetDateTime>> completionDateOperations;
```